screens/HomeAuth.js

```diff
@@ -8,13 +8,14 @@ import {
 } from 'react-native';
 import { Button } from 'react-native-elements';
 import * as Facebook from 'expo-facebook';
-import { CONFIG } from '../config.js';
+import CONFIG from '../config.js';
 
 const splashScreenImage = require('../assets/heymentorsplash.png');
 
 const API_URL = CONFIG.ENV === 'PROD' ? CONFIG.API_URL : CONFIG.TEST_API_URL;
 
 class HomeAuth extends Component {
+  // @TODO: look into https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md to clear the issue
```

Comment:
I think this should either be ignored here or the rule be removed because the rule is forcing `navigationOptions` to be sandwiched between the other functions rather than entered as an initial static variable